### PR TITLE
Made Changes in README and Contributing sections

### PR DESCRIPTION
**Pull request by aflah02**
ISSUE #50 

fixes #

I've fixed some grammatical errors and moved the contribution related section to CONTRIBUTING file

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**
NA

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,6 +18,145 @@ Then, please [create a new issue in the GitHub repository][new-issue] describing
 
 Be sure to include as much detail as possible including step-by-step descriptions, specific examples, screenshots or mockups, and reasoning for why the enhancement might be worthwhile.
 
+## 📌 Setting up Locale Environment
+
+### 🚩 Forking repository :
+
+1. Fistly you have to make your own copy of project. For that you have to fork the repository. You can find the fork button on the top-right side of the browser window. (Refer image below )
```
Comment (kaiwalyakoparkar) [inline CONTRIBUTING.md:25]: Change "Fistly" to "Firstly"

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (aflah02) [reply CONTRIBUTING.md:25]: Ohkk will do

Comment (kaiwalyakoparkar) [reply CONTRIBUTING.md:25]: and link issue no #50 to it

**Comment by aflah02** [comment]
Done @kaiwalyakoparkar

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM

**Comment by kaiwalyakoparkar** [comment]
It's conflicting again. Please follow the steps again!

**Comment by aflah02** [comment]
@kaiwalyakoparkar I'm sorry I can't redo this anymore this is the third time, please assign someone else if they are interested

**Comment by kaiwalyakoparkar** [comment]
Alright!

**Comment by aflah02** [comment]
@kaiwalyakoparkar Yeah I understand thanks for helping along the way

**Comment by kaiwalyakoparkar** [comment]
@aflah02 thanks for holding till now. You can always contribute to other issues and don't forget to mention the mentors in the comments. I think they might be busy with some other work. that's why this has happened. Have a good day!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,6 +18,145 @@ Then, please [create a new issue in the GitHub repository][new-issue] describing
...
+
+1. `git branch` will list all the branched in the repository
+2. Now type `git branch <your-branch-name>`
+3. You can check the created branch by `git branch`
+4. But still if you start editing the edits will go to the main branch. To change the path of the edits type in `git checkout <your-branch-name>`
+5. Now you are ready to do the desired changes.
+
+## 📌 Setting up project using `npm` :
+
+1. Open this clonned folder in text editor of your choice
```
Comment (suubh) [inline CONTRIBUTING.md:77]: It should be Cloned. 
```suggestion
1. Open this clonned folder in text editor of your choice
```

Comment (aflah02) [reply CONTRIBUTING.md:77]: I am no longer working on this I've closed this thanks for your time I believe it has been resolved :)

**Pull request closed without merging**